rust/ql/lib/codeql/rust/frameworks/Reqwest.qll

```diff
@@ -0,0 +1,16 @@
+/**
+ * Provides modeling for the `reqwest` library.
+ */
+
+private import rust
+private import codeql.rust.Concepts
+
+/**
+ * A call to `reqwest::get` or `reqwest::blocking::get`.
+ */
+private class ReqwestGet extends RemoteSource::Range {
+  ReqwestGet() {
+    this.asExpr().getExpr().(CallExpr).getExpr().(PathExpr).getPath().getResolvedPath() =
+      ["crate::get", "crate::blocking::get"]
```

Comment:
I would have expected this to be `reqwest::get` instead of `crate::get`. Those `crate` prefixes seem rather unhelpful. This is not a fault of this PR, just something we ought to address. @redsun82 what do you think?

Reply:
see https://github.com/github/codeql/pull/18069#discussion_r1854209768 :)

Reply:
Yep, I agree, and raised the point in https://github.com/github/codeql/pull/18069#discussion_r1853603742 .  There's now a partial resolution.